data/sql_updates/create_reports_presidential_view.sql

```diff
@@ -33,8 +33,6 @@ select
     items_on_hand_liquidated as items_on_hand_liquidated,
     loans_received_from_cand_per as loans_received_from_candidate_period,
     loans_received_from_cand_ytd as loans_received_from_candidate_ytd,
-    net_contb_sum_page_per as net_contributions_period,
-    net_op_exp_sum_page_per as net_operating_expenditures_period,
```

Comment:
We want to take the sums out of totals, but I think we should leave them in all the reports, since this is data reported by the committee and we do display it in the summaries.